src/Lumi/Components/Spacing.purs

```diff
@@ -44,3 +47,6 @@ toPixels S88 = "88px"
 toPixels S96 = "96px"
 toPixels S104 = "104px"
 toPixels S112 = "112px"
+
+instance isStylePropertySpace :: IsStyleProperty Space where
+  prop = str <<< toPixels
```

Comment:
I really like this!

Reply:
Also, on a side note, would you mind adding a `S40` constructor to the `Space` type above? It seems we forgot to add that one when this module was created 😅